html/run3.html

```diff
@@ -30,6 +30,20 @@ <h5>Run 3 sample planning</h5>
             <td>{{row[2]}}</td>
           </tr>
         {% endfor %}
+	<!-- Space for free text inclusion -->  
+        <tr>
+          <td> <textarea style="width: 100%; margin-bottom: 8px;" placeholder="Add an identifier for the sample, e.g. name" id="datasetnamerun3">  </textarea> </td>
+          <td> <textarea style="width: 100%; margin-bottom: 8px;" placeholder="Add the wished number of events" id="numberofevents"> </textarea> </td>
+          <td> <textarea style="width: 100%; margin-bottom: 8px;" placeholder="Add the interested PWGs; if many, add them separated by a comma" id="pwginterested"> </textarea> </td>
+        </tr>
+        <tr>
+          <td> </td>
```

Comment:
Look into "colspan" attribute of `<td>`.
You can also try to refresh the page in the script, this might work: `window.location.reload(true);`.

Reply:
I think I don't understand this comment..should I change:
<td> 

to 

< td colspan="2"> ?

Reply:
You can do colspan="3" to have a only one cell in the row and somehow align button to the right. It does not change much, just visuals.